trestle/core/commands/author/ssp.py

```diff
@@ -354,12 +358,13 @@ def _get_implementations(self, control_file: pathlib.Path) -> List[ossp.Implemen
                 by_comp: ossp.ByComponent = gens.generate_sample_model(ossp.ByComponent)
                 by_comp.description = text
                 statement: ossp.Statement = gens.generate_sample_model(ossp.Statement)
-                part_label = section.split(' ')[1].replace('.', '')
-                statement_label = f'{control_id}_smt.{part_label}'
-                statement.by_components = {statement_label: by_comp}
+                # the following may be needed with the 1.0.0 changes
+                # part_label = section.split(' ')[1].replace('.', '')  # noqa: E800
+                # statement_label = f'{control_id}_smt.{part_label}'   # noqa: E800
+                statement.by_components = [by_comp]
```

Comment:
Add issue to track this - related to this NIST parameter substitution issue.

Reply:
done